jetty-osgi/pom.xml

```diff
@@ -12,7 +12,7 @@
   <packaging>pom</packaging>
 
   <properties>
-    <osgi-version>3.6.0.v20100517</osgi-version>
+    <osgi-version>3.7.1</osgi-version>
     <osgi-services-version>3.2.100.v20100503</osgi-services-version>
```

Comment:
This `osgi-services-version` is still using ...

``` xml
<dependency>
  <groupId>org.eclipse.osgi</groupId>
  <artifactId>org.eclipse.osgi.services</artifactId>
  <version>3.2.100.v20100503</version>
</dependency>
```

Which is the old, bad, maven coordinates. (which is a 10 year old artifact now with no updates on this coordinate space)
Link to old / bad coordinates: https://search.maven.org/artifact/org.eclipse.osgi/org.eclipse.osgi.services

The correct coordinates for osgi.services is ...

``` xml
<dependency>
  <groupId>org.eclipse.platform</groupId>
  <artifactId>org.eclipse.osgi.services</artifactId>
  <version>3.9.0</version>
</dependency>
```

Which was updated 14-Sep-2020
Link to proper coordinates: https://search.maven.org/artifact/org.eclipse.platform/org.eclipse.osgi.services

Reply:
If you're positive that those 2 different coordinates are for the osgi services, then I'll try updating the version of that. I did look for a newer version, but obviously only at the old coordinates.

Reply:
The version from the bad coordinates is also dependent on the older osgi version ...

``` xml
<dependency>
      <groupId>org.eclipse.osgi</groupId>
      <artifactId>org.eclipse.osgi</artifactId>
      <version>3.6.0.v20100517</version>
    </dependency>
```

Reply:
The listed packages for both ...

*Old Coordinate Space*

```
$ jar -tvf org.eclipse.osgi.services-3.2.100.v20100503.jar | cut -c37- | sed -e "s/[A-Za-z0-9\$]*\.class//" | sort -u | grep -v "META-INF"
org/
org/osgi/
org/osgi/service/
org/osgi/service/cm/
org/osgi/service/component/
org/osgi/service/device/
org/osgi/service/event/
org/osgi/service/http/
org/osgi/service/io/
org/osgi/service/log/
org/osgi/service/metatype/
org/osgi/service/provisioning/
org/osgi/service/upnp/
org/osgi/service/useradmin/
org/osgi/service/wireadmin/
```

*New Coordinate Space*

```
$ jar -tvf org.eclipse.osgi.services-3.9.0.jar | cut -c37- | sed -e "s/[A-Za-z0-9\$\-]*\.class//" | sort -u | grep -v "META-INF"
org/
org/osgi/
org/osgi/service/
org/osgi/service/cm/
org/osgi/service/component/
org/osgi/service/component/annotations/
org/osgi/service/component/annotations/packageinfo
org/osgi/service/component/runtime/
org/osgi/service/component/runtime/dto/
org/osgi/service/device/
org/osgi/service/event/
org/osgi/service/http/
org/osgi/service/http/context/
org/osgi/service/http/runtime/
org/osgi/service/http/runtime/dto/
org/osgi/service/http/whiteboard/
org/osgi/service/log/
org/osgi/service/metatype/
org/osgi/service/provisioning/
org/osgi/service/upnp/
org/osgi/service/useradmin/
org/osgi/service/wireadmin/
```